### PR TITLE
Upload fix

### PR DESCRIPTION
**Pull request by mitjade**

### CODE REVIEW

File: packages/server/src/modules/upload/resolvers.js [hunk 1]
```diff
@@ -27,7 +35,7 @@ const storeFS = ({ stream, filename }) => {
 };
 
 const processUpload = async uploadPromise => {
-  const { stream, filename: name, mimetype: type, encoding } = await uploadPromise;
```
Comment (mitjade) [inline packages/server/src/modules/upload/resolvers.js]: For some reason `filename` and `mimetype` are undefined?

Comment (vlasenko) [reply packages/server/src/modules/upload/resolvers.js]: @mitjade Maybe it should be vice versa, e.g. `name: filename`?

Comment (mitjade) [reply packages/server/src/modules/upload/resolvers.js]: I just removed the renaming and it works ok, just not sure why this is not working. If `uploadPromise` returns `name` and you want to assign it to `filename`?

Comment (vlasenko) [reply packages/server/src/modules/upload/resolvers.js]: It have returned `filename` previously and I needed `name` from it. And it returned `mimetype` and I needed `type` instead to keep backward compatibility with old upload namings

Comment (mitjade) [reply packages/server/src/modules/upload/resolvers.js]: So is it ok now as is?

Comment (vlasenko) [reply packages/server/src/modules/upload/resolvers.js]: I think yes

**Pull request merged**